api/src/main/java/org/eclipse/microprofile/jwt/config/Names.java

```diff
@@ -40,4 +40,10 @@
      * The expected iss claim value to validate against an MP-JWT.
      */
     String ISSUER = "mp.jwt.verify.issuer";
+    
+    /**
+     * The expected "aud" claim value(s), separated by commas. 
```

Comment:
This is not exactly the same as in the spec doc. Here it appears that "aud" claim value must match exactly with the contents of the value of the property.

To avoid confusing, harmonize the texts.

Reply:
fixed, I think

Reply:
@rdebusscher resolving, LGTM, please reopen if you feel an extra tweak is needed

Reply:
@rdebusscher LGTM, reopen if needed